binding/flutter/lib/porcupine_error.dart

```diff
@@ -9,49 +9,84 @@
 // specific language governing permissions and limitations under the License.
 //
 
-class PvAudioException implements Exception {
+// ignore_for_file: unnecessary_brace_in_string_interps
```

Comment:
I think if you just use `$code` instead of `${code}` you don't have to ignore it